README.md

```diff
@@ -85,7 +85,9 @@ You will need to follow steps below to install required platform and also optimi
         "horizon": 5,
         "ft_value": 1.5,
         "itb_value": 0.2,
+        "decay_base": 0.84,
```

Comment:
Did the decay_base get removed? can't see it in the regurlar_settings.json anymore